### PR TITLE
[13.0][ADD] account_security_extra

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
- Change the menu position "Invoicing/Configuration/Chart of Accounts" to "Invoicing/Accounting/Chart of Accounts"
- In group Show Full Accounting Features adds write and create permission to account.account

### CODE REVIEW

**Comment by ChristianSantamaria** [comment]
@dalonsod can u review this?

File: account_security_extra/views/account_security_extra_menu.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account.menu_action_account_form" model="ir.ui.menu">
+        <!-- Overriden: account.account_account_menu -->
```
Comment (dalonsod) [inline account_security_extra/views/account_security_extra_menu.xml:5]: Could you make the same uninstall hook as #65 for this undo this change when addon is removed?

File: account_security_extra/security/account_security_extra.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account.access_account_account" model="ir.model.access">
+        <field name="perm_write" eval="1"/>
+        <field name="perm_create" eval="1"/>
+    </record>
```
Comment (dalonsod) [inline account_security_extra/security/account_security_extra.xml]: Is it possible replacing this update with a new ACL that adds this functionality? I don't know if two ACLs to the same model and group could live together... If not, we should add to uninstall hook getting back original permissions

File: account_security_extra/security/ir.model.access.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_account_user,account.account user,account.model_account_account,base.group_user,1,1,1,0
```
Comment (dalonsod) [inline account_security_extra/security/ir.model.access.csv]: Group too permissive, and not group mentioned in module `DESCRIPTION`, better this one (read permission is actually available, see  https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/addons/account/security/ir.model.access.csv#L11, but it could be kept here anyway)
```suggestion
access_account_account_user,account.account user,account.model_account_account,account.group_account_user,1,1,1,0
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Only one last thing to be fixed

**Review by dalonsod — APPROVED** [review]
Tested :+1:
